lib/lrama/states.rb

```diff
@@ -92,6 +92,20 @@ def compute
       report_duration(:compute_default_reduction) { compute_default_reduction }
     end
 
+    def compute_ielr
+      report_duration(:split_states) { split_states }
+      report_duration(:compute_direct_read_sets) { compute_direct_read_sets }
```

Comment:
[note]: Compute LA set again, once states are splitted. See Phase 4 and Phase 5 on "3.1. Overview".